platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -1242,7 +1246,10 @@ def get_transceiver_threshold_info(self):
                              ]
         transceiver_dom_threshold_info_dict = dict.fromkeys(dom_info_dict_keys, 'N/A')
 
-        if self.sfp_type == OSFP_TYPE:
+        if not self.dom_supported:
+            return transceiver_dom_threshold_info_dict
```

Comment:
if dom is not supported what is the expected values in the info dic? hard coded values aligned to what?

Reply:
Now sure whether I understood the question correctly. 
if dom is not supported, all fields should be `N/A`. this is also what we did for SFP ports.